opm/parser/eclipse/EclipseState/Schedule/WellTracerProperties.hpp

```diff
@@ -0,0 +1,41 @@
+/*
+  Copyright 2018 NORCE.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef WELLTRACERPROPERTIES_HPP_HEADER_INCLUDED
+#define WELLTRACERPROPERTIES_HPP_HEADER_INCLUDED
+#include <vector>
+#include <string>
+#include <map>
+
+namespace Opm {
+
+    struct WellTracerProperties {
+        std::map< std::string, double > m_tracerProperties;
```

Comment:
As long as the data container is accessed through `get` and `set` methods I would suggest `WellTracerProperties` was made into a class, and that the `m_tracerProperties` member was made private.

You have called the member `tracerProperties` but it seems it only holds concentrations? I.e. maybe:
```
std::map<std::string,double> tracer_concentrations;
```
would be a better choice?

Reply:
Good suggestions, they are implemented.